boefjes/boefjes/dependencies/encryption.py

```diff
@@ -34,8 +34,8 @@ def __init__(self, private_key: str, public_key: str):
 
     def encode(self, contents: str) -> str:
         encrypted_contents = self.box.encrypt(contents.encode())
-        encrypted_contents = base64.b64encode(encrypted_contents)
-        return encrypted_contents.decode()
+        base64_encrypted_contents = base64.b64encode(encrypted_contents)
+        return base64_encrypted_contents.decode()
```

Comment:
This effectively doubles the used memory, what is the size of the contents here?

Reply:
I don't really see how this doubles memory. Changing the name of the variable doesn't change which Python objects are created.  That encrypted_contents can be garbage collected after the return statement instead of after the assignment but that shouldn't make much difference.

The contents is also small, because this is for encryption of katalogus database. This is also not turned on by default. I don't think anyone uses it except for the installation at iRealisatie, because it isn't really documented as far as I know and when it was broken we only heard feedback from them about it.